### PR TITLE
Update workflow executions tables

### PR DESCRIPTION
**Pull request by deepsidhu85**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR updates the workflow executions tables to reorder the columns, and the state column has been updated to display badges. The summary page for a workflow execution has also been updated to display the optional name if set

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

image

image

image

image

image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Run a workflow execution
2. View the workflow executions table (global) and verify the state column is now the 3rd column and has a viral pill
3. Run an automated workflow execution from a project
4. View the project workflow executions table and verify the state column is now the 3rd column and has a viral pill
5. View the summary page for a workflow execution. If the name was set then it should be in the listing

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

**Review by ericenns — APPROVED** [review]
Looks good to me!

File: app/views/workflow_executions/_workflow_execution.html.erb [hunk 1]
```diff
@@ -2,17 +2,26 @@
...
 >
-  <td class="px-6 py-4 whitespace-nowrap sticky left-0 bg-white dark:bg-gray-800 z-10">
+  <td
+    class="
+      px-6
+      py-4
+      whitespace-nowrap
+      sticky
+      left-0
+      bg-white
```
Comment (joshsadam) [inline app/views/workflow_executions/_workflow_execution.html.erb:12]: **nitpick:** Can you update the background colour on this column to be `bg-slate-50 dark:bg-slate-900`, that is how I am trying to get all primary columns that have scroll to be.

File: app/views/projects/workflow_executions/_workflow_execution.html.erb [hunk 2]
```diff
@@ -25,9 +25,7 @@
     },
     class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
```
Comment (joshsadam) [inline app/views/projects/workflow_executions/_workflow_execution.html.erb:26]: **nitpick:** Can up update the first column as I said previously with `bg-slate-50 dark:bg-slate-900`

File: app/views/workflow_executions/_workflow_execution.html.erb (hunk 1 above)
Comment (deepsidhu85) [reply app/views/workflow_executions/_workflow_execution.html.erb]: Updated in 11361028a12f6069f80e16afbc750c6028be7147

File: app/views/projects/workflow_executions/_workflow_execution.html.erb (hunk 2 above)
Comment (deepsidhu85) [reply app/views/projects/workflow_executions/_workflow_execution.html.erb:26]: Updated in 11361028a12f6069f80e16afbc750c6028be7147

**Review by joshsadam — APPROVED** [review]
Looks good to me

**Pull request merged**